.circleci/config.yml

```diff
@@ -42,7 +42,7 @@ jobs:
           command: yarn test
       - run:
           name: run integration tests
-          command: yarn test:integration
+          command: cd integration && yarn && yarn integration
```

Comment:
Is this change related? 

Seems to be the same as the [yarn test:integration](https://github.com/aerogear/graphback/blob/master/package.json#L35) script.

Reply:
actually that should have been reverted in 349272c - thanks.